src/anubhav/StringOperations/FrequencyOfCharacters.java

```diff
@@ -0,0 +1,47 @@
+package anubhav.StringOperations;
+
+import java.util.Scanner;
+
+/*Find the frequency of character from a given String array.
+Note: Take String and character from the user.
+Input : technocredits
+char : e
+output : e -> 2*/
+
+public class FrequencyOfCharacters {
+
+int count;
+
+	String originalString;
+	char targetCh;
+	void frequency() {
+		count=0;
```

Comment:
you can create local variable not instance variable. method should be paramterised